third_party/move/move-bytecode-verifier/src/code_unit_verifier.rs

```diff
@@ -43,9 +43,233 @@ impl<'a> CodeUnitVerifier<'a> {
             .map_err(|e| e.finish(Location::Module(module.self_id())))
     }
 
+    fn pattern_check_for_pack(code_opt: Option<&CodeUnit>) -> PartialVMResult<()> {
+        if code_opt.is_none() {
```

Comment:
Use `let Some(code) = code_opt else {};` instead of the raw unwrap later.

Reply:
done